docs/adrs/adr-001-external-tooling.md

```diff
@@ -0,0 +1,53 @@
+# ADR 1: External Tooling
+
+Brian Cavalier,
+David Chase,
+Tylor Steinberger <tlsteinberger167@gmail.com>,
+Frederik Krautwald <fkrautwald@gmail.com>,
+
+## Status
+
+**Proposed** | ~~Accepted~~ | ~~Deprecated~~ | ~~Superceded~~
```

Comment:
I changed it to "Superseded" [in the template PR](https://github.com/mostjs/core/pull/218/commits/d6df06ace1fb24e2ec3b0231a673813069e79e1c).  Let's update it here, too.